src/Symfony/Component/Intl/README.md

```diff
@@ -1,11 +1,7 @@
 Intl Component
 =============
 
-A PHP replacement layer for the C intl extension that also provides access to
-the localization data of the ICU library.
-
-The replacement layer is limited to the locale "en". If you want to use other
```

Comment:
the info it is en-only is missing in `symfony/polyfill-intl-icu` for the migration though

Reply:
fixed in https://github.com/symfony/polyfill/commit/692afa35beb6beb91058fc1e2d44c9f849852966